lib/logstash/pluginmanager/util.rb

```diff
@@ -1,64 +1,101 @@
+module LogStash::PluginManager
 
-class LogStash::PluginManager::Util
-
-  def self.logstash_plugin?(gem)
+  # check for valid logstash plugin gem name & version or .gem file, logs errors to $stdout
+  # uses Rubygems API and will remotely validated agains the current Gem.sources
+  # @param plugin [String] plugin name or .gem file path
+  # @param version [String] gem version requirement string
+  # @return [Boolean] true if valid logstash plugin gem name & version or a .gem file
+  def self.is_logstash_plugin?(plugin, version = nil)
+    if is_plugin_file?(plugin)
```

Comment:
same here, prefer not using `is_`; this is exampled by JRuby's java bridging implicitly making `isFoo` available as `foo?`

```
>> java.lang.String.new.empty?
=> true
```